config/accounts.go

```diff
@@ -6,4 +6,19 @@ type Account struct {
 	Disabled      bool        `mapstructure:"disabled" json:"disabled"`
 	CacheTTL      DefaultTTLs `mapstructure:"cache_ttl" json:"cache_ttl"`
 	EventsEnabled bool        `mapstructure:"events_enabled" json:"events_enabled"`
+	GDPR          AccountGDPR `mapstructure:"gdpr" json:"gdpr"`
+}
+
+// AccountGDPR represents account-specific GDPR configuration
+type AccountGDPR struct {
+	Enabled            *bool                  `mapstructure:"enabled" json:"enabled,omitempty"`
+	IntegrationEnabled AccountGDPRIntegration `mapstructure:"integration_enabled" json:"integration_enabled"`
+}
+
+// AccountGDPRIntegration indicates whether GDPR is enabled for each request type
+type AccountGDPRIntegration struct {
+	AMP   *bool `mapstructure:"amp"   json:"amp,omitempty"`
```

Comment:
Why is there a mapstructure on `AccountGDPR` but not `AccountGDPRIntegration`? Is the `mapstructure` required if there's already a json attribute? I really don't know. Just looking to reduce duplication if we can.

Reply:
The `mapstructure` isn't needed so I removed it.